### ISSUE TITLE
Static assets such as screenshots

### ISSUE THREAD

**Issue by jsonmaur** [timberio/docs#3]
Currently all static assets used in the docs (such as screenshots) are hosted on contentful. As we move our documentation away from contentful, where should the assets be hosted?

**Comment by jsonmaur** [comment]
How would we go about uploading and linking to a screenshot to be used in a documentation page?

**Comment by zsherman** [comment]
Local assets need to be supported no matter what.

**Comment by binarylogic** [comment]
Agree, if there are local assets those should upload with the final build so that can be relatively referenced.
